### PR TITLE
django 1.10 fix

### PR DESCRIPTION
**Pull request by waprin**
@theromis hope you don't mind but I just wrote the tests for you, good call on reusing to_python. Also added to the docs that we are only supporting versions of Django that Django supports.

### CODE REVIEW

File: oauth2client/contrib/django_orm.py [hunk 1]
```diff
@@ -16,6 +16,12 @@
 
 Utilities for using OAuth 2.0 in conjunction with
 the Django datastore.
+
+Note that only supported versions of Django are supported
```
Comment (theacodes) [inline oauth2client/contrib/django_orm.py]: This sentence reads a little funny, how about just "Only Django versions 1.8+ are supported."?

File: oauth2client/contrib/django_util/__init__.py [hunk 2]
```diff
@@ -20,6 +20,11 @@
 if the user has authorized, and helper shortcuts to create the authorization
 URL otherwise.
 
+Note that only supported versions of Django are supported
```
Comment (theacodes) [inline oauth2client/contrib/django_util/__init__.py]: Same here.

File: oauth2client/contrib/django_orm.py (hunk 1 above)
Comment (waprin) [inline oauth2client/contrib/django_orm.py]: done

File: oauth2client/contrib/django_util/__init__.py (hunk 2 above)
Comment (waprin) [inline oauth2client/contrib/django_util/__init__.py]: done

**Comment by nathanielmanistaatgoogle** [comment]
Tell me about the scary flaming red `cla: no` label on this pull request?

**Comment by dhermes** [comment]
@nathanielmanistaatgoogle It's because the Google CLA checker doesn't like PRs by committee (see the "committed with"):

screen_shot_001

**Comment by jonparrott** [comment]
Yeah it's just googlebot being obtuse. @waprin just needs to confirm things are good.

**Pull request merged**